network/player.go

```diff
@@ -35,16 +38,56 @@ func ConvertToGetPlayerResponse(p model.Player) GetPlayerResponse {
 			ID:   p.ID,
 			Name: p.Name,
 		},
-		Games: convertToParticipatingGames(p.Games),
 	}
 }
 
-func convertToParticipatingGames(mgs map[model.GameID]model.PlayerColor) map[model.GameID]string {
-	games := make(map[model.GameID]string, len(mgs))
-	for g, c := range mgs {
-		games[g] = c.String()
+type ActiveGame struct {
+	PlayerNamesByID  map[model.PlayerID]string `json:"players"`
+	PlayerColorsByID map[model.PlayerID]string `json:"colors"`
+	Created          time.Time                 `json:"created"`
+	LastMove         time.Time                 `json:"lastMove"`
+}
+
+type GetActiveGamesForPlayerResponse struct {
+	Player      Player                      `json:"player"`
+	ActiveGames map[model.GameID]ActiveGame `json:"activeGames"`
```

Comment:
This answers my question above ☝️, is there any reason this needs to be a map, or could it be a slice?

Reply:
slice makes sense ➕ 1